int8[] motor_cmd